### ISSUE TITLE
As the organizer of the service "Badia bus" (on-demand service pilot in val Badia), I would like to see the real-time data of the service being integrated and made available through the Open Data Hub

### ISSUE THREAD

**Issue by rcavaliere** [noi-techpark/bdp-commons#613]

**Comment by dulvui** [comment]
@rcavaliere I will reactivate the data collector this sprint

**Comment by rcavaliere** [comment]
@dulvui did you reconnect the Data Collector to the new end-point? The service should have started and we should see it on mobility.meran.eu if data is coming. Let me know!

**Comment by dulvui** [comment]
@rcavaliere yes but only in testing and should be visible here https://journey.opendatahub.testingmachine.eu but I was not able to find the stations.
Should I deploy to production?

**Comment by rcavaliere** [comment]
@dulvui the service should be visible in the Municipality of Badia / Abtei. I don't see data there. But if I see our API (e.g. https://mobility.api.opendatahub.testingmachine.eu/v2/flat,node/ON_DEMAND_STOP) I see new data here! Maybe there is something that we should configure on the application side so to see this data as well?

**Comment by dulvui** [comment]
@rcavaliere I checked now and the OTP uses the production endpoint of the mobility API also in testing.
I will change it now to use the testing mobility API so we can test.

**Comment by dulvui** [comment]
@rcavaliere Now the on demand buses are visible on testing. There was a problem with the `smetadata.groups`, since some stops of badia don't have a group assigned and so the otp crashed. For now I set the group to -1, if none is set.
https://journey.opendatahub.testingmachine.eu/

**Comment by rcavaliere** [comment]
@dulvui wonderful, thanks for the fix! I think we put the Data Collector in production and release this on the official applications, like mobility.meran.eu

**Comment by dulvui** [comment]
@rcavaliere Now it is live in production and visible on https://mobility.meran.eu/#/

**Comment by rcavaliere** [comment]
@dulvui good, but I see a strange effect now, see screenshot... can you check why we have this? 

![Screenshot from 2023-07-06 14-12-27](https://github.com/noi-techpark/bdp-commons/assets/9825179/87f2bfb3-7846-497c-a6c4-10df712d7469)

**Comment by dulvui** [comment]
@rcavaliere I will investigate what causes this problem, seems some problem with the otp calculations.
